src/niscope/metadata/enums.py

```diff
@@ -876,35 +870,42 @@
     'TriggerCoupling': {
         'values': [
             {
-                'name': 'AC',
+                'name': 'NISCOPE_VAL_AC',
                 'value': 0,
 'documentation': {
 'description': 'AC coupling',
 },
             },
             {
-                'name': 'DC',
+                'name': 'NISCOPE_VAL_DC',
                 'value': 1,
 'documentation': {
 'description': 'DC coupling',
 },
             },
             {
-                'name': 'HF_REJECT',
+                'name': 'NISCOPE_VAL_HF_REJECT',
                 'value': 2,
 'documentation': {
 'description': 'Highpass filter coupling',
 },
             },
             {
-                'name': 'LF_REJECT',
+                'name': 'NISCOPE_VAL_LF_REJECT',
                 'value': 3,
 'documentation': {
 'description': 'Lowpass filter coupling',
 },
             },
             {
-                'name': 'AC_PLUS_HF_REJECT',
+                'name': 'LF_REJECT',
```

Comment:
Is this a known bug? Or not a bug?
NISCOPE_VAL_LF_REJECT and LF_REJECT?

Reply:
This is a bug and has already been CARed. We overwrite this whole enum using enum_addon